flow_transferFiles2Apollo/dataTransfer-blat.cwl

```diff
@@ -0,0 +1,26 @@
+#!usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+requirements:
+  - class: InlineJavascriptRequirement
+
+baseCommand: [scp]
```

Comment:
Can you change this command from `scp -r` to `rsync -avP`?

Reply:
I changed scp to rsync -vzP.
If the problem of option -a is solved, I will change it to rsync -avP.